lib/libswan/constants.c

```diff
@@ -2108,13 +2108,15 @@ static const char *const v2_notification_status_name[] = {
 
 static const char *const v2_notification_private_name[] = {
 #define S(E) [E - v2N_NULL_AUTH] = #E
-	S(v2N_NULL_AUTH),	/* 40960, used for mixed OE */
```

Comment:
Unfortunately this isn't so easy.  The result will be an array containing `50209 - 40960 + 1` elements, oops.
There are two ways to handle this:

- create multiple enum_name tables and chain them together
- convert this to a sparse_name table which contains name+value pairs

the choice is yours

Reply:
Thanks for catching this. Fixed in 25fa57e.